### ISSUE TITLE
Onboarding Sven as a cloud.gov Pages Operator

### ISSUE THREAD

**Issue by apburnes** [cloud-gov/product#2365]
#### User Story

 ## Special Notes

- **Do not create this issue until the System Owner has formally authorized and requested it.** You can obtain that OK by one of two ways:
  A:
  - ~A: System Owner creates this issue~

  B:
  - [X] B.1: System owner emails cloud-gov-compliance@gsa.gov and cloud-gov-pages-operations@gsa.gov with their authorization
  - [X] B.2: An operator adds links to the email archive of the authorizing email. 

---

Onboard Sven

#### Background (Optional)

https://docs.google.com/document/d/1tS-9hLb2yHtp500N3obmu0X70XpIMATuPlov2_vDVS0/edit

#### Acceptance Criteria
- [x] Share README
- [x] Grant appropriate access to Federalist backend in cloud.gov as a CF Org Manager
- [x] Add to federalist-admins
- [x] Add to federalist-admins in federalist-users org.
- [x] Add to cloud-gov-team in cloud-gov org.
- [x] Add to pages-ops in cloud-gov org.
- [x] Make owner in the federalist-users org.
- [x] Add to #federalist- channels in Slack as needed, and add to @federalist-operators Slack user group
- [x] Add to daily standup, retro, and sprint planning
- [x] Ensure user has Federalist Github project and brief on its use
- [x] Ensure user has GPG signing set up for GitHub: https://docs.google.com/document/d/11UDxvfkhncyLEs-NUCniw2u54j4uQBqsR2SBiLYPUZc/edit
- [ ] ~Add to New Relic~ (Turned off for SCR)
- [x] Add to federalist-support, federalist-alerts, and federalist-inquiries Google Groups as needed
- [x] Add them as an agent to https://federalist-support.zendesk.com/agent
- [x] add to ~Federalist~ cloud.gov search.gov account
- [x] Add new user as developer to the gsa-18f-federalist cloud.gov cloudfoundry organization
- [ ] ~Add to statuspage.io account~ (blocked by license availability)
- [x] Have a cloud.gov operator add to the “pages.admin” group in the cloud.gov UAA by using the make-pages-admin.sh script
- [x] Have a cloud.gov operator add to the “concourse.pages” team in the cloud.gov OpsUAA by using the make-pages-ops-admin.sh script so they can access Concourse CI
- [x] Add to Docker Hub
- [x] Add to AWS (Production GovCloud only)
- [ ] ~Add to `FederalistLocal` Github org and `federalist-local-admins` team~ (Not needed for working on Pages)
- [x] Add to cloud.gov calendar

**Comment by svenaas** [comment]
The checklist and the [background doc](https://docs.google.com/document/d/1tS-9hLb2yHtp500N3obmu0X70XpIMATuPlov2_vDVS0/edit#) do not precisely agree. On the presumption that the checklist here is more up to date I've added a couple of suggestions in the doc and tagged @apburnes over there, but if there's someone else that it would be better to tag, please let me know. I do note that the doc is version numbered (currently 1.8).

**Comment by apburnes** [comment]
Added to the Pages cloud.gov cloud foundry org as an Org Manger.

**Comment by svenaas** [comment]
I have access to Federalist (https://federalist.18f.gov/) in Search.gov, but I think that's it. Is there a separate Site defined in there for Pages, or do they share the configuration?

**Comment by svenaas** [comment]
I have access to the correct Search site now, and have done everything I can do except the updated AWS setup, which is next.

**Comment by apburnes** [comment]
🥳
